server/src/responders/user-responders.js

```diff
@@ -335,6 +336,24 @@ async function requestAccessResponder(
   await sendAccessRequestEmailToAshoat(request);
 }
 
+const friendRequestInputValidator = tShape({
+  userIDs: t.list(t.String),
+});
+
+async function friendRequestCreationResponder(
+  viewer: Viewer,
+  input: any,
+): Promise<void> {
+  const request: AddFriendsRequest = input;
+  await validateInput(viewer, friendRequestInputValidator, request);
```

Comment:
One thing we haven't talked about is the experience of receiving a friend request. I think it should pop up as a new thread, since the app is based on this assumption that all notifs come from threads.

Probably worth doing in a separate PR later. [Here's a link](https://github.com/Ashoat/squadcal/blob/master/server/src/bots/app-version-update.js#L62-L67) to some existing code that does something similar.

I also just created #50 to track the more general question of what a thread with a non-friend looks like.